specs/specs.js

```diff
@@ -243,5 +243,40 @@ describe("Fingerprint2", function () {
         });
       });
     });
+
+    describe("preprocessor", function () {
+      it("checks if user_agent not processed by default", function (done) {
```

Comment:
This description should not mention user agent. It should mention preprocessor instead. You're not testing something specific to the user agent field. You're testing the preprocessor option.